akka-actor/src/main/resources/reference.conf

```diff
@@ -420,6 +420,7 @@ akka {
         mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
+      # deprecated (because relies on blocking)
```

Comment:
Right, that's something I was wondering about... Some of the mailboxes do have a timeout parameter - those we want to keep then?